src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -24,44 +22,28 @@ type PropsType = {
   cities: Array<CityModel>,
   navigateToDashboard: (city: CityModel) => void,
   t: TFunction,
-  theme: ThemeType
+  theme: ThemeType,
+  location: LocationType,
+  proposeNearbyCities: boolean,
+  tryAgain: () => Promise<void>
```

Comment:
This prop seems to be unused

Reply:
Nope, it gets passed to `CitySelector` by `{...this.props}`